### PR TITLE
Added rudimentary arangodb support to mantl-universe

### PR DESCRIPTION
**Pull request by emdem**
The uninstall needs work, in case there are multiple instances installed.

I am able to uninstall/reinstall cleanly on the same cluster, over and over, which is a start.

Edit:
I'm not surprised I missed some issues - I was testing by copying/pasting into consul kv via the interface. I have a fork of mantl-api, I should just setup a docker hub image so that I can do a better job testing end to end properly, instead of copy/pasting via the consul UI.

### CODE REVIEW

File: repo/packages/A/arangodb/1/mantl.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  "mesos": {
+    "master": "zk://zookeeper.service.consul:2181/mesos"
+  },
+  "arangodb": {
+    "id": "arangodb",
+    "volume-path": "/data/arangodb",
+    "mesos-authenticate":"{{mantl.mesos.authentication-enabled}}"
+    {{#mantl.mesos.authentication-enabled}},
+    "principal":"{{mantl.mesos.principal}}",
+    "secret": "{{mantl.mesos.secret}}"{{/mant.mesos.authentication-enabled}}
```
Comment (ryane) [inline repo/packages/A/arangodb/1/mantl.json]: small typo here: `mant.mesos.authentication-enabled` should be `mantl.mesos.authentication-enabled`

File: repo/packages/A/arangodb/1/mantl.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "mesos": {
+    "master": "zk://zookeeper.service.consul:2181/mesos"
+  },
+  "arangodb": {
+    "id": "arangodb",
```
Comment (ryane) [inline repo/packages/A/arangodb/1/mantl.json:6]: I think we need to add an `arangodb.zk` property here. In https://github.com/mesosphere/universe/blob/version-1.x/repo/packages/A/arangodb/1/config.json, the default is `zk://master.mesos:2181/arangodb`. So, we probably need to set it to `zk://zookeeper.service.consul:2181/arangodb`

**Comment by ryane** [comment]
looking good. I ran into a couple of issues when installing that I added notes about above. Let me know if those make sense.

Also, is there any reason to support the 0 version? Since the 1 version works and supports auth, I am not sure we need to include the 0 version at all. If we remove the 0 files, mantl-api will just consider that version "unsupported".

File: repo/packages/A/arangodb/1/uninstall.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "zookeeper": {
+    "delete": [
+      {
+        "path": "/arangodb",
```
Comment (ryane) [inline repo/packages/A/arangodb/1/uninstall.json]: I _think_ we could do something like

``` json
"path": "/arangodb/{{arangodb.id}}"
```

here to support removing just a specific instance of the framework but we would need to test it

File: repo/packages/A/arangodb/1/mantl.json (hunk 2 above)
Comment (emdem) [inline repo/packages/A/arangodb/1/mantl.json:6]: Added it - should be working now.

**Comment by ryane** [comment]
thanks @emdem!

**Pull request merged**